### PR TITLE
adding appveyor

### PR DESCRIPTION
**Pull request by patjouk**
Fix MozillaFoundation/mofo-devops#514

First try at adding appveyor to Brackets.

### CODE REVIEW

**Comment by patjouk** [comment]
I've created the project associated to it on Appveyor under the mozilla foundation organization. I'm waiting on more info about how to setup tests and will update this ticket as soon as I know more.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+environment:
+  nodejs_version: "6"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install
+
+
+test: false
```
Comment (cadecairos) [inline .appveyor.yml]: can you add a newline at the end of the file?

**Comment by patjouk** [comment]
Looking at travis config file, I saw that `grunt build-browser-compressed` was started as test. I tried to use it in appveyor but I'm getting the same error than when I tried to make brackets runs on my mac:

```
Tracing dependencies for: main
Error: ENOENT: no such file or directory, open 'C:\projects\brackets\src\thirdparty\text\text.js'
In module tree:
    brackets
      language/LanguageManager
        file/FileUtils
          utils/Global
```
[full log here](https://ci.appveyor.com/project/mozillafoundation/brackets/build/10#L1256).

Should I remove the tests for now and open a ticket about that error or try to fix it now?

**Comment by patjouk** [comment]
TIL: submodule! I realized that `--recursive` was passed will doing a `git clone` in the readme. I've search for more info about that and added `git submodule update --recursive --init` in the install section. It now works.

This PR is now ready for review.

**Pull request merged**